tensorflow/contrib/cmake/make.bat

```diff
@@ -0,0 +1,109 @@
+@echo off
```

Comment:
This is a highly specific script that will only work on your machine, with all the tools installed in the locations you installed them.
Please remove and expand README.md in this folder.

Reply:
+1. Can you please remove this file?

Reply:
Alright, removed @mrry @gunan